integration-tests/code-generator-tests/src/test/java/com/squareup/anvil/test/Utils.kt

```diff
@@ -0,0 +1,6 @@
+package com.squareup.anvil.test
+
+import kotlin.reflect.KClass
+
+internal infix fun KClass<*>.extends(other: KClass<*>): Boolean =
+  other.java.isAssignableFrom(this.java)
```

Comment:
brb copying this into all of my projects because I can never remember the correct operand order for `isAssignableFrom`

Reply:
Same :-D